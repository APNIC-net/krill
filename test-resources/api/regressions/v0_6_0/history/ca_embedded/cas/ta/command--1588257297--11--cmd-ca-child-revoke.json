{
  "actor": "krill",
  "time": "2020-04-30T14:34:57.207052Z",
  "handle": "ta",
  "version": 17,
  "sequence": 11,
  "details": {
    "child_revoke_key": [
      "child",
      {
        "class_name": "0",
        "key": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85"
      }
    ]
  },
  "effect": {
    "events": [
      17,
      18,
      19
    ]
  }
}