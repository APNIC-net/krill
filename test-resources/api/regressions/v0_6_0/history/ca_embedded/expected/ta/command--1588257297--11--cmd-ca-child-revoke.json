{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:34:57.207052Z",
    "handle": "ta",
    "version": 17,
    "sequence": 11,
    "details": {
      "child_revoke_key": [
        "child",
        {
          "class_name": "0",
          "key": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85"
        }
      ]
    },
    "effect": {
      "events": [
        17,
        18,
        19
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "ta",
        "version": 17,
        "details": {
          "child_key_revoked": [
            "child",
            "0",
            "C4CF00E42E046EBE5275589D3462F1B1F43FBA85"
          ]
        }
      },
      {
        "id": "ta",
        "version": 18,
        "details": {
          "object_set_updated": [
            "0",
            {
              "3E1358DAB62425AC531554CF43AF8420979B9476": {
                "number": 5,
                "revocations_delta": {
                  "added": [
                    {
                      "serial": "503598915590458265774719609317104819039126994854",
                      "expires": "2021-04-30T14:34:56Z"
                    },
                    {
                      "serial": "324437802174706850658552259052205978732316029692",
                      "expires": "2020-05-01T14:34:56.120728Z"
                    }
                  ],
                  "dropped": []
                },
                "manifest_info": {
                  "name": "3E1358DAB62425AC531554CF43AF8420979B9476.mft",
                  "current": {
                    "content": "MIILEgYJKoZIhvcNAQcCoIILAzCCCv8CAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQUYDzIwMjAwNDMwMTQyOTU3WhgPMjAyMDA1MDExNDM0NTdaBglghkgBZQMEAgEwUzBRFiwzRTEzNThEQUI2MjQyNUFDNTMxNTU0Q0Y0M0FGODQyMDk3OUI5NDc2LmNybAMhAGLG4nti4GZCDOc1WFR3j/71eSxhBCTd1tpg5YVlvIlOoIIImTCCCJUwggd9oAMCAQICFBqCXNPRZr0aOH4y5ABobnR3+1u5MA0GCSqGSIb3DQEBCwUAMIICLTGCAikwggIlBgNVBAMTggIcMzA4MjAxMEEwMjgyMDEwMTAwREYyNDM1NDk2MjgyOTA0QTI4NTgzRkU4NjY0MDA2N0ZBMERFNDQxQzNGQ0ZBNkVCNzkxREM1QUM5RkU5QTkwRjAxQTYzMkQxNDhDMjIyQTAwQ0EyRDdDMkY1NUNCMkIyMUI0REY3Q0JFN0YwNzA1NEE5ODk2RUY3RjcxNEE3MUY2QTY3Nzg2OEJFMDU0QUJFMzVGNENBNTU3NTUzN0M1OEM0NDBEQ0QwOTdGQjg0RkU2NzRFQ0RBOUU2MTE5Mjk5M0NCMDU0ODhCNEQ2NDFDMjAxREIzN0YxRUIxNUMyMjBDMEJEQ0U2RUNCMjdGM0VDMjg0QjAyMDkxMkQzNDA1QjU0QzI2MjZBODc1RThGQ0Y5ODgzREU2QjcxMjE1NURBNTQ2NkZENTU3MjMxNzZEQUM0MDQwOEEyNkUzRTI0QTNBMDA2NUE4QTI5NjA4NDE4MTA5RUQzMzFCNEIzODcyQTJGRDkyNjJERjg3NjgwNUU2RkY3MEUyOEZDNTAxRjIyRDVERDY4MDlEODkyOTIzNEU0MzZCOEFFMUY3MDUzMEUyNzYwRkYzQkZDQTk0OEYzQ0U5RTZCQTdBMUVCQUFBRTUyNjNENENFOUFBMTA1OTNBNERFMzdEQUMxNDg3RUVFNDZBNkNCQjZEODQwMkY4REUyMUFFQkY1NUYzMTg5MTcwMjAzMDEwMDAxMB4XDTIwMDQzMDE0Mjk1N1oXDTIwMDUwNzE0MzQ1N1owggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBDRjE3OUNCNTM1RUM4NjU5N0YyOTNDRkY1N0ExOTVEN0I3ODRGOUQ0NDE3ODU2M0NEREI0ODYzOTY1NjVDOTg1MTI0OUZENTZDNTJDMjdEODFEMEYwMTQ3M0NCMEI0RTNGRDI1RUNBOEQ3QTE1NTgyOUIzNTJCQzlFNjIwRDc0NTJCN0E0OEYyOTU1NDFCQTExMUUxRTFFNTEzQkZDQTMwNjQzNkJBMEZCN0MyRjJGMDhFRjQ4OTk1MTg4MUE0NDA4RkJFQ0U5NTVEQkM2RkIzNEVCMEI4ODIzRjZEMjQ4RkNGREU3Qzc3NjNGQ0VENUY3ODJDQTFFNzhFRTVBNTBENkVFRjE3OEI5MzMwQTY1M0ZBMEYyRUFCQTNBRkRERjU4MUVGNzY2M0MwRjY2MjRFOUNEMTgxQUUzQjkyRjE3Nzc4ODg0QkE5QTA1NzQwN0U1NjdGNzg0QjRBRkE4RjUyNUVBNjY3NjFCQTEwOUQxMjQzMDEwMTZFMDIzQ0FBQjc0QjNBRTY2OUNBQzJEREVBMjczMTYzOUJBRTM5QjAzQ0Y4N0ZDNTVBMzVGNUM5MjY3QkM4RTJBQTg5QTdFMkU4RDRCQjNCNUIyOTBCMEVCODNBNkU0OTI2NUUzNDBBMTE1NDg0QUQ0Njc2RkJGRDQ3QUE5N0NFNzY5RTkyNkVCNTAyMDMwMTAwMDEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDPF5y1NeyGWX8pPP9XoZXXt4T51EF4VjzdtIY5ZWXJhRJJ/VbFLCfYHQ8BRzywtOP9Jeyo16FVgps1K8nmINdFK3pI8pVUG6ER4eHlE7/KMGQ2ug+3wvLwjvSJlRiBpECPvs6VXbxvs06wuII/bSSPz958d2P87V94LKHnjuWlDW7vF4uTMKZT+g8uq6Ov3fWB73ZjwPZiTpzRga47kvF3eIhLqaBXQH5Wf3hLSvqPUl6mZ2G6EJ0SQwEBbgI8qrdLOuZpysLd6icxY5uuObA8+H/FWjX1ySZ7yOKqiafi6NS7O1spCw64Om5JJl40ChFUhK1Gdvv9R6qXznaekm61AgMBAAGjggGnMIIBozAdBgNVHQ4EFgQURpdFETdVo05Yf9K+Y0Kx1sknwPAwHwYDVR0jBBgwFoAUPhNY2rYkJaxTFVTPQ6+EIJeblHYwDgYDVR0PAQH/BAQDAgeAMFkGA1UdHwRSMFAwTqBMoEqGSHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvMC8zRTEzNThEQUI2MjQyNUFDNTMxNTU0Q0Y0M0FGODQyMDk3OUI5NDc2LmNybDA8BggrBgEFBQcBAQQwMC4wLAYIKwYBBQUHMAKGIHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvdGEuY2VyMGQGCCsGAQUFBwELBFgwVjBUBggrBgEFBQcwC4ZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzNFMTM1OERBQjYyNDI1QUM1MzE1NTRDRjQzQUY4NDIwOTc5Qjk0NzYubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQB/89CS7W0cFS0k3DhIUmcPstC6APl421Un2DVCf1GOdF88ckrfLReFXqljKDNjqDY8JyKFTurdhXFaLgsYejgn3jEkHkDDOo7ZmnJCpxr60VG4Iu2kRNmXML0tgrhGncowR5FS/HlZFqNyi9xzbij4JKD0oxdwJfgaUkVWkoZf34iZVRjIdXUUCYd+quBlswNY1YZ3g2BS+61mit6TZtDX6g3PQ0ia0fPgGE0FPetucgg3++EkgKUkzWf4XwpHVYF/rkxuzfXWQN9n0z3AEvI2RccmIfHFqQSZku1CEDBvI/z0DIk3yAdVj8CmtC4Xx97v8hwtEblbeZktfxMkd2XzMYIBrDCCAagCAQOAFEaXRRE3VaNOWH/SvmNCsdbJJ8DwMA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MzAxNDM0NTdaMC8GCSqGSIb3DQEJBDEiBCDK0xw0dJ+a6lFniDNFgghHCU903u0e/K8hKc4O1QXmODANBgkqhkiG9w0BAQEFAASCAQAzAg04hYgUn/nBZWqfwbE+L7VY7yV/URWT7lCoIMkWu/lwSSm60hc0MrekAY5bO3bDSOfmhTh8rljNpTAop2RL/E16lAoUsegwXUJY6FKr3XhsjRk/UTU8L8hD2VT1NhLFjvCWjSaSTg1pzxJlG/cJTMV1H8wHezSFtvhc/T0aFFHViduovrc7jmktpz/8kCvMFTup+ZjlpUPXTenKdfndipBQ7QV5JlhYHAZms6Y+fjhfjQzdfPeTkz5e3cpgRQPB6M/59FrZJVMQN04xm9dKmVU09+S49I+4n+lBaZ/jYNAUvje4+tPRgpyMJ5Lezb+rDe7CcS09zTGLZ2g6izrv",
                    "serial": "151340943325456051750617675288054826231125466041",
                    "expires": "2020-05-01T14:34:57.211411Z"
                  },
                  "next_update": "2020-05-01T14:34:57.211411Z",
                  "old": "61bf0cfc86b87fe7c93555f1b451bc949959b6b26a3074ad0f5f584bc16eb83c"
                },
                "crl_info": {
                  "name": "3E1358DAB62425AC531554CF43AF8420979B9476.crl",
                  "current": {
                    "content": "MIIEvzCCA6cCAQEwDQYJKoZIhvcNAQELBQAwggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBERjI0MzU0OTYyODI5MDRBMjg1ODNGRTg2NjQwMDY3RkEwREU0NDFDM0ZDRkE2RUI3OTFEQzVBQzlGRTlBOTBGMDFBNjMyRDE0OEMyMjJBMDBDQTJEN0MyRjU1Q0IyQjIxQjRERjdDQkU3RjA3MDU0QTk4OTZFRjdGNzE0QTcxRjZBNjc3ODY4QkUwNTRBQkUzNUY0Q0E1NTc1NTM3QzU4QzQ0MERDRDA5N0ZCODRGRTY3NEVDREE5RTYxMTkyOTkzQ0IwNTQ4OEI0RDY0MUMyMDFEQjM3RjFFQjE1QzIyMEMwQkRDRTZFQ0IyN0YzRUMyODRCMDIwOTEyRDM0MDVCNTRDMjYyNkE4NzVFOEZDRjk4ODNERTZCNzEyMTU1REE1NDY2RkQ1NTcyMzE3NkRBQzQwNDA4QTI2RTNFMjRBM0EwMDY1QThBMjk2MDg0MTgxMDlFRDMzMUI0QjM4NzJBMkZEOTI2MkRGODc2ODA1RTZGRjcwRTI4RkM1MDFGMjJENURENjgwOUQ4OTI5MjM0RTQzNkI4QUUxRjcwNTMwRTI3NjBGRjNCRkNBOTQ4RjNDRTlFNkJBN0ExRUJBQUFFNTI2M0Q0Q0U5QUExMDU5M0E0REUzN0RBQzE0ODdFRUU0NkE2Q0JCNkQ4NDAyRjhERTIxQUVCRjU1RjMxODkxNzAyMDMwMTAwMDEXDTIwMDQzMDE0Mjk1N1oXDTIwMDUwMTE0MzQ1N1owggERMCUCFEgjlfHIinCnlZ89JUTbNxU0jkwiFw0yMDA1MDExNDM0NTJaMCUCFH/1TNl4T2PeGSVc3mW9ujFmgtbrFw0yMTA0MzAxNDM0NTRaMCUCFBmOl615WUVaTyFwTFaRJrdEJHxAFw0yMDA1MDExNDM0NTRaMCUCFCtMq+B2QGUze6COvCH4UoH2miSsFw0yMTA0MzAxNDM0NTVaMCUCFDnhOHoO5NZ1jK9qYfGG9t7Pqnu/Fw0yMDA1MDExNDM0NTVaMCUCFFg2KAjkXFtnEakqC7gpLdYZUL+mFw0yMTA0MzAxNDM0NTZaMCUCFDjUS1EfuQTrj/oNgvR+BGsHK3L8Fw0yMDA1MDExNDM0NTZaoC8wLTAfBgNVHSMEGDAWgBQ+E1jatiQlrFMVVM9Dr4Qgl5uUdjAKBgNVHRQEAwIBBTANBgkqhkiG9w0BAQsFAAOCAQEAjATNsZ6/2MXbyZ0KggaqhpaFbBCiYDEgW+zBJxdENMZgeJMWLYHbv1qBOqP+tb3dKkyS+n5PwqHgdyHilHiGLKR+zJNAfyIyUjzgzMrHBGjCSkNXaneN/MWIXSr8BF+BFChQAXLaVSnsBe/Y6G6W6b1Dfryoe3HAIjJPFQ/cBjIZDt6AFxOFQiK9D6jQtImR5YIteN5nYVBBUlvFaIAf5i+Fo/QCZdd205qsgJxxSq54yHHFfyvr1wQo2QOfVIIG2liZIaqo1hIK17DcbHYXPuZr8z0pjJ05sESUE14UgkzQ81dgqAx40XuJcB1pcsfqanVvb8KjDJ1MhavlNKsSEA==",
                    "serial": "5",
                    "expires": "2020-05-01T14:34:57.207643Z"
                  },
                  "old": "f23c6c94b5c45b17786ce8cce542c67b59111f2453a0c7549ac84942d7fccfd2"
                },
                "objects_delta": {
                  "ca_repo": "rsync://localhost/repo/ta/0",
                  "added": [],
                  "updated": [
                    {
                      "name": "3E1358DAB62425AC531554CF43AF8420979B9476.crl",
                      "object": {
                        "content": "MIIEvzCCA6cCAQEwDQYJKoZIhvcNAQELBQAwggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBERjI0MzU0OTYyODI5MDRBMjg1ODNGRTg2NjQwMDY3RkEwREU0NDFDM0ZDRkE2RUI3OTFEQzVBQzlGRTlBOTBGMDFBNjMyRDE0OEMyMjJBMDBDQTJEN0MyRjU1Q0IyQjIxQjRERjdDQkU3RjA3MDU0QTk4OTZFRjdGNzE0QTcxRjZBNjc3ODY4QkUwNTRBQkUzNUY0Q0E1NTc1NTM3QzU4QzQ0MERDRDA5N0ZCODRGRTY3NEVDREE5RTYxMTkyOTkzQ0IwNTQ4OEI0RDY0MUMyMDFEQjM3RjFFQjE1QzIyMEMwQkRDRTZFQ0IyN0YzRUMyODRCMDIwOTEyRDM0MDVCNTRDMjYyNkE4NzVFOEZDRjk4ODNERTZCNzEyMTU1REE1NDY2RkQ1NTcyMzE3NkRBQzQwNDA4QTI2RTNFMjRBM0EwMDY1QThBMjk2MDg0MTgxMDlFRDMzMUI0QjM4NzJBMkZEOTI2MkRGODc2ODA1RTZGRjcwRTI4RkM1MDFGMjJENURENjgwOUQ4OTI5MjM0RTQzNkI4QUUxRjcwNTMwRTI3NjBGRjNCRkNBOTQ4RjNDRTlFNkJBN0ExRUJBQUFFNTI2M0Q0Q0U5QUExMDU5M0E0REUzN0RBQzE0ODdFRUU0NkE2Q0JCNkQ4NDAyRjhERTIxQUVCRjU1RjMxODkxNzAyMDMwMTAwMDEXDTIwMDQzMDE0Mjk1N1oXDTIwMDUwMTE0MzQ1N1owggERMCUCFEgjlfHIinCnlZ89JUTbNxU0jkwiFw0yMDA1MDExNDM0NTJaMCUCFH/1TNl4T2PeGSVc3mW9ujFmgtbrFw0yMTA0MzAxNDM0NTRaMCUCFBmOl615WUVaTyFwTFaRJrdEJHxAFw0yMDA1MDExNDM0NTRaMCUCFCtMq+B2QGUze6COvCH4UoH2miSsFw0yMTA0MzAxNDM0NTVaMCUCFDnhOHoO5NZ1jK9qYfGG9t7Pqnu/Fw0yMDA1MDExNDM0NTVaMCUCFFg2KAjkXFtnEakqC7gpLdYZUL+mFw0yMTA0MzAxNDM0NTZaMCUCFDjUS1EfuQTrj/oNgvR+BGsHK3L8Fw0yMDA1MDExNDM0NTZaoC8wLTAfBgNVHSMEGDAWgBQ+E1jatiQlrFMVVM9Dr4Qgl5uUdjAKBgNVHRQEAwIBBTANBgkqhkiG9w0BAQsFAAOCAQEAjATNsZ6/2MXbyZ0KggaqhpaFbBCiYDEgW+zBJxdENMZgeJMWLYHbv1qBOqP+tb3dKkyS+n5PwqHgdyHilHiGLKR+zJNAfyIyUjzgzMrHBGjCSkNXaneN/MWIXSr8BF+BFChQAXLaVSnsBe/Y6G6W6b1Dfryoe3HAIjJPFQ/cBjIZDt6AFxOFQiK9D6jQtImR5YIteN5nYVBBUlvFaIAf5i+Fo/QCZdd205qsgJxxSq54yHHFfyvr1wQo2QOfVIIG2liZIaqo1hIK17DcbHYXPuZr8z0pjJ05sESUE14UgkzQ81dgqAx40XuJcB1pcsfqanVvb8KjDJ1MhavlNKsSEA==",
                        "serial": "5",
                        "expires": "2020-05-01T14:34:57.207643Z"
                      },
                      "old": "f23c6c94b5c45b17786ce8cce542c67b59111f2453a0c7549ac84942d7fccfd2"
                    },
                    {
                      "name": "3E1358DAB62425AC531554CF43AF8420979B9476.mft",
                      "object": {
                        "content": "MIILEgYJKoZIhvcNAQcCoIILAzCCCv8CAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQUYDzIwMjAwNDMwMTQyOTU3WhgPMjAyMDA1MDExNDM0NTdaBglghkgBZQMEAgEwUzBRFiwzRTEzNThEQUI2MjQyNUFDNTMxNTU0Q0Y0M0FGODQyMDk3OUI5NDc2LmNybAMhAGLG4nti4GZCDOc1WFR3j/71eSxhBCTd1tpg5YVlvIlOoIIImTCCCJUwggd9oAMCAQICFBqCXNPRZr0aOH4y5ABobnR3+1u5MA0GCSqGSIb3DQEBCwUAMIICLTGCAikwggIlBgNVBAMTggIcMzA4MjAxMEEwMjgyMDEwMTAwREYyNDM1NDk2MjgyOTA0QTI4NTgzRkU4NjY0MDA2N0ZBMERFNDQxQzNGQ0ZBNkVCNzkxREM1QUM5RkU5QTkwRjAxQTYzMkQxNDhDMjIyQTAwQ0EyRDdDMkY1NUNCMkIyMUI0REY3Q0JFN0YwNzA1NEE5ODk2RUY3RjcxNEE3MUY2QTY3Nzg2OEJFMDU0QUJFMzVGNENBNTU3NTUzN0M1OEM0NDBEQ0QwOTdGQjg0RkU2NzRFQ0RBOUU2MTE5Mjk5M0NCMDU0ODhCNEQ2NDFDMjAxREIzN0YxRUIxNUMyMjBDMEJEQ0U2RUNCMjdGM0VDMjg0QjAyMDkxMkQzNDA1QjU0QzI2MjZBODc1RThGQ0Y5ODgzREU2QjcxMjE1NURBNTQ2NkZENTU3MjMxNzZEQUM0MDQwOEEyNkUzRTI0QTNBMDA2NUE4QTI5NjA4NDE4MTA5RUQzMzFCNEIzODcyQTJGRDkyNjJERjg3NjgwNUU2RkY3MEUyOEZDNTAxRjIyRDVERDY4MDlEODkyOTIzNEU0MzZCOEFFMUY3MDUzMEUyNzYwRkYzQkZDQTk0OEYzQ0U5RTZCQTdBMUVCQUFBRTUyNjNENENFOUFBMTA1OTNBNERFMzdEQUMxNDg3RUVFNDZBNkNCQjZEODQwMkY4REUyMUFFQkY1NUYzMTg5MTcwMjAzMDEwMDAxMB4XDTIwMDQzMDE0Mjk1N1oXDTIwMDUwNzE0MzQ1N1owggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBDRjE3OUNCNTM1RUM4NjU5N0YyOTNDRkY1N0ExOTVEN0I3ODRGOUQ0NDE3ODU2M0NEREI0ODYzOTY1NjVDOTg1MTI0OUZENTZDNTJDMjdEODFEMEYwMTQ3M0NCMEI0RTNGRDI1RUNBOEQ3QTE1NTgyOUIzNTJCQzlFNjIwRDc0NTJCN0E0OEYyOTU1NDFCQTExMUUxRTFFNTEzQkZDQTMwNjQzNkJBMEZCN0MyRjJGMDhFRjQ4OTk1MTg4MUE0NDA4RkJFQ0U5NTVEQkM2RkIzNEVCMEI4ODIzRjZEMjQ4RkNGREU3Qzc3NjNGQ0VENUY3ODJDQTFFNzhFRTVBNTBENkVFRjE3OEI5MzMwQTY1M0ZBMEYyRUFCQTNBRkRERjU4MUVGNzY2M0MwRjY2MjRFOUNEMTgxQUUzQjkyRjE3Nzc4ODg0QkE5QTA1NzQwN0U1NjdGNzg0QjRBRkE4RjUyNUVBNjY3NjFCQTEwOUQxMjQzMDEwMTZFMDIzQ0FBQjc0QjNBRTY2OUNBQzJEREVBMjczMTYzOUJBRTM5QjAzQ0Y4N0ZDNTVBMzVGNUM5MjY3QkM4RTJBQTg5QTdFMkU4RDRCQjNCNUIyOTBCMEVCODNBNkU0OTI2NUUzNDBBMTE1NDg0QUQ0Njc2RkJGRDQ3QUE5N0NFNzY5RTkyNkVCNTAyMDMwMTAwMDEwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDPF5y1NeyGWX8pPP9XoZXXt4T51EF4VjzdtIY5ZWXJhRJJ/VbFLCfYHQ8BRzywtOP9Jeyo16FVgps1K8nmINdFK3pI8pVUG6ER4eHlE7/KMGQ2ug+3wvLwjvSJlRiBpECPvs6VXbxvs06wuII/bSSPz958d2P87V94LKHnjuWlDW7vF4uTMKZT+g8uq6Ov3fWB73ZjwPZiTpzRga47kvF3eIhLqaBXQH5Wf3hLSvqPUl6mZ2G6EJ0SQwEBbgI8qrdLOuZpysLd6icxY5uuObA8+H/FWjX1ySZ7yOKqiafi6NS7O1spCw64Om5JJl40ChFUhK1Gdvv9R6qXznaekm61AgMBAAGjggGnMIIBozAdBgNVHQ4EFgQURpdFETdVo05Yf9K+Y0Kx1sknwPAwHwYDVR0jBBgwFoAUPhNY2rYkJaxTFVTPQ6+EIJeblHYwDgYDVR0PAQH/BAQDAgeAMFkGA1UdHwRSMFAwTqBMoEqGSHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvMC8zRTEzNThEQUI2MjQyNUFDNTMxNTU0Q0Y0M0FGODQyMDk3OUI5NDc2LmNybDA8BggrBgEFBQcBAQQwMC4wLAYIKwYBBQUHMAKGIHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvdGEuY2VyMGQGCCsGAQUFBwELBFgwVjBUBggrBgEFBQcwC4ZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzNFMTM1OERBQjYyNDI1QUM1MzE1NTRDRjQzQUY4NDIwOTc5Qjk0NzYubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQB/89CS7W0cFS0k3DhIUmcPstC6APl421Un2DVCf1GOdF88ckrfLReFXqljKDNjqDY8JyKFTurdhXFaLgsYejgn3jEkHkDDOo7ZmnJCpxr60VG4Iu2kRNmXML0tgrhGncowR5FS/HlZFqNyi9xzbij4JKD0oxdwJfgaUkVWkoZf34iZVRjIdXUUCYd+quBlswNY1YZ3g2BS+61mit6TZtDX6g3PQ0ia0fPgGE0FPetucgg3++EkgKUkzWf4XwpHVYF/rkxuzfXWQN9n0z3AEvI2RccmIfHFqQSZku1CEDBvI/z0DIk3yAdVj8CmtC4Xx97v8hwtEblbeZktfxMkd2XzMYIBrDCCAagCAQOAFEaXRRE3VaNOWH/SvmNCsdbJJ8DwMA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MzAxNDM0NTdaMC8GCSqGSIb3DQEJBDEiBCDK0xw0dJ+a6lFniDNFgghHCU903u0e/K8hKc4O1QXmODANBgkqhkiG9w0BAQEFAASCAQAzAg04hYgUn/nBZWqfwbE+L7VY7yV/URWT7lCoIMkWu/lwSSm60hc0MrekAY5bO3bDSOfmhTh8rljNpTAop2RL/E16lAoUsegwXUJY6FKr3XhsjRk/UTU8L8hD2VT1NhLFjvCWjSaSTg1pzxJlG/cJTMV1H8wHezSFtvhc/T0aFFHViduovrc7jmktpz/8kCvMFTup+ZjlpUPXTenKdfndipBQ7QV5JlhYHAZms6Y+fjhfjQzdfPeTkz5e3cpgRQPB6M/59FrZJVMQN04xm9dKmVU09+S49I+4n+lBaZ/jYNAUvje4+tPRgpyMJ5Lezb+rDe7CcS09zTGLZ2g6izrv",
                        "serial": "151340943325456051750617675288054826231125466041",
                        "expires": "2020-05-01T14:34:57.211411Z"
                      },
                      "old": "61bf0cfc86b87fe7c93555f1b451bc949959b6b26a3074ad0f5f584bc16eb83c"
                    }
                  ],
                  "withdrawn": [
                    {
                      "name": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85.cer",
                      "hash": "b40eea29fb6507454ecfa26aad5367f6af65f2bd9d94990e838408d602f17f1b"
                    }
                  ]
                }
              }
            }
          ]
        }
      },
      {
        "id": "ta",
        "version": 19,
        "details": {
          "child_certificates_updated": [
            "0",
            {
              "issued": [],
              "removed": [
                "C4CF00E42E046EBE5275589D3462F1B1F43FBA85"
              ]
            }
          ]
        }
      }
    ]
  }
}